### ISSUE TITLE
CSS code issue

### ISSUE THREAD

**Issue by toudidel** [theme-materialized/hexo-theme-materialized#2]
I use hexo plugin [hexo-filter-mathjax](https://github.com/stevenjoezhang/hexo-filter-mathjax) for formatting mathematical formulas and expressions.

There is strange thing when using theme `hexo-theme-materialized`.
Some CSS code is inserted before the excerpt of the post. That issue I can see only in that theme.

Example URL: https://akademia.coderion.pl/tags/binarny/

**Comment by code004Accepted** [comment]
Hi. Thanks for your kind reply.

However, it seems that the problem in the example url does not appear on my computer. The screenshots are listed below:






(The image didn't appear in the second screenshot because of the low Internet speed)

**Comment by toudidel** [comment]
I cannot see your screenshot in the message.
I paste my screenshot: https://pasteboard.co/IDpA0nr.png

**Comment by code004Accepted** [comment]
![39e9351c96431e7d3906e085fa29a682.png](https://i.loli.net/2019/10/25/k175xoTqLwivMyG.png)

It seems that there are no problem?

I'm using Google Chrome in Windows 10.

**Comment by toudidel** [comment]
You are correct of course. Strange, but I need some investigation in that case.
